example/example.c

```diff
@@ -0,0 +1,20 @@
+#include "example.h"
+
+#define FUNCTION_INDEX_GROUP_EXAMPLE_SIZE 2
+
+static void **function_index_group_example;
```

Comment:
I'd advocate replacing `void **function_index_group_example` by the equivalent `void *function_index_group_example[]`, which more closely matches your intention here (the latter reads as "an array of pointers" whereas the former reads as "a pointer to a pointer").

Reply:
I think we went with `static void **` because we *assign* to that variable name below the value of `functions` which is a `void **`.  I think if we instead use `static void **`, then we can't do the simple assignment statement below.

The error is:
```
example.c:10:34: error: array type 'void *[]' is not assignable
    function_index_group_example = functions;
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
```

Reply:
Fair enough.